ROMFS/px4fmu_common/init.d/airframes/4601_droneblocks_dexi_5

```diff
@@ -43,6 +43,7 @@ param set-default CA_ROTOR3_PX -0.0762
 param set-default CA_ROTOR3_PY 0.09525
 param set-default CA_ROTOR3_KM -0.05
 
+param set-default EKF2_HGT_REF 2
```

Comment:
Range sensor height reference probably isn't a good idea.

Reply:
Dropped the commit.